docs/Microservices/overview.md

```diff
@@ -84,6 +84,28 @@ For more information about power states supported by the REST APIs, see [Intel®
 
 Intel® AMT enables remote management of a device, even when the OS isn't running, through KVM over IP support. No additional equipment is needed for this feature.  With KVM control, IT administrators can access and update PCs and devices as if they were onsite. It eliminates the need for remote KVM switches and other hardware. 
 
+## Passwords
+
+There are four levels of passwords: 
+
+1. **Intel® Manageability Engine BIOS Extensions (MEBX) Password:** Intel® MEBX enables the configuration of the Intel Manageability Engine (ME) platform. The Intel® MEBX password along with other configuration details is set on the managed device in the BIOS. For details about entering the Intel® MEBX BIOS settings, see [MEBX Unprovision](../Topics/MEBX/unprovision.md).
```

Comment:
I think we need to keep this high-level and specific to the password we are discussing.  Something like:
"This password is used to secure the local Intel(r) Manageability Engine BIOS Extensions (MEBx) menu.  This password is only used when physically accessing the managed device during system boot (Ctrl-P Menu on most devices)."  I think any additional links we put here should be more general in nature in regards to MEBx.

Reply:
This is exactly the kind of feedback I need. Thanks.